src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -148,6 +148,13 @@ void NonClientIslandWindow::OnSize(const UINT width, const UINT height)
         return;
     }
 
+    // TODO: Currently, when we enter fullscreen mode, we draw the border around

```

Comment:
TODO is waiting for #3064's work, but is this solved in #3064 or does it need another item filed to follow up?